src/write.c

```diff
@@ -941,6 +941,30 @@ static avifCodecType avifEncoderGetCodecType(const avifEncoder * encoder)
     return avifCodecTypeFromChoice(encoder->codecChoice, AVIF_CODEC_FLAG_CAN_ENCODE);
 }
 
+static avifBool avifEncoderDataForceKeyframeForAlpha(const avifEncoderData * data, const avifEncoderItem * colorItem, avifAddImageFlags addImageFlags)
```

Comment:
(1) Nit: Document this function. One reason is that the function name sounds like an action, so I thought this function would force a keyframe for alpha and the boolean return value indicates success/failure.

Alternatively, add "Should" or "NeedTo" to the function name?

(2) Another useful piece of info to document, which cannot be inferred from the function name, is when this function is called.

This function is called after we have encoded a frame of the color channel and before we encode the corresponding frame of the alpha channel.

Reply:
Done.